src/olympia/editors/helpers.py

```diff
@@ -718,6 +718,7 @@ def confirm_auto_approved(self):
         is incrementing AddonApprovalsCounter, which also resets the last
         human review date to now, and log it so that it's displayed later
         in the review page."""
+        self.version = self.addon.current_version
```

Comment:
Yeah it's not ideal... I wasn't happy about that either, so that makes 2 of us, I'll change it. In a previous commit I already added the possibility for `log_action()` to take a version, so that should be enough.